src/services/cm-service.ts

```diff
@@ -18,55 +21,85 @@ class ApiError extends CLIError {
 }
 
 export class CmService {
+  private static debug = debugFactory(CmService)
   private readonly cmApi: CmAPI
 
   constructor() {
     const cmUrl = process.env.ENTANDO_CLI_ECR_URL
     const cmToken = process.env.ENTANDO_CLI_ECR_TOKEN
 
     if (!cmUrl || !cmToken) {
-      throw new CLIError('Both "process.env.ENTANDO_CLI_ECR_URL" and "process.env.ENTANDO_CLI_ECR_TOKEN" should have values')
+      throw new CLIError(
+        'Environment variables "ENTANDO_CLI_ECR_URL" and "ENTANDO_CLI_ECR_TOKEN" should have values'
+      )
     }
 
     this.cmApi = new CmAPI(cmUrl, cmToken)
   }
 
   public async getBundles(): Promise<CmBundle[]> {
+    CmService.debug(`fetching bundles from ${process.env.ENTANDO_CLI_ECR_URL}`)
     try {
       const response = await this.cmApi.getBundles()
       return response.data.payload
     } catch (error) {
-      const errorMsg = 'Failed to get bundles'
-      throw new ApiError(errorMsg, error as AxiosError | Error)
+      throw new ApiError('Failed to get bundles', error as AxiosError | Error)
     }
   }
 
   public async getBundleMicroservices(bundleId: string): Promise<Plugin[]> {
+    CmService.debug(
```

Comment:
@zonia3000 Probably better to move those debug statemets inside client in order to print the full request URL